concent_api/core/message_handlers.py

```diff
@@ -954,6 +958,58 @@ def store_message(
     return stored_message
 
 
+def handle_send_subtask_results_verify(
+    request,
+    subtask_results_verify: message.concents.SubtaskResultsVerify
+):
+    report_computed_task = subtask_results_verify.subtask_results_rejected.report_computed_task
+    compute_task_def = report_computed_task.task_to_compute.compute_task_def
+    validate_id_value(
```

Comment:
Please add validation for:
1. task_id
2. task_to_compute

Reply:
done;
'task_id' is checked within `validate_golem_message_task_to_compute(r` -> I have added:
`validate_golem_message_task_to_compute(report_computed_task.task_to_compute)`